### PR TITLE
Some improvements: audio and conanfile.py

### PR DESCRIPTION
**Pull request by czoido**
Partly taken from: https://github.com/froglets/flappy-frog/pull/13

### CODE REVIEW

File: .github/workflows/_build.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ on:
       conanfile:
         type: string
         required: false
-        default: conanfile.txt
+        default: conanfile.py
```
Comment (jgsogo) [inline .github/workflows/_build.yml:37]: Do not change defaults, pass the proper value: https://github.com/czoido/flappy-frog/pull/1

```suggestion
        default: conanfile.txt
```

File: .github/workflows/_build_macos.yml [hunk 2]
```diff
@@ -38,7 +38,7 @@ on:
       conanfile:
         type: string
         required: false
-        default: conanfile.txt
+        default: conanfile.py
```
Comment (jgsogo) [inline .github/workflows/_build_macos.yml:41]: Do not change defaults, pass the proper value: https://github.com/czoido/flappy-frog/pull/1

```suggestion
        default: conanfile.txt
```

**Comment by jgsogo** [comment]
conflicts...

**Comment by danimtb** [comment]
There are some apparently legit errors reported by the CI

File: src/Game.cpp [hunk 3]
```diff
@@ -8,15 +8,16 @@
 #include <fmt/core.h>
 #include <SDL.h>
 #include <SDL_image.h>
+#include <SDL_mixer.h>
 
 
 // Callback function for button state changed callback
 void cb_button_state_changed(uint8_t state, void* user_data) {
-    Frog* frog = (Frog*)user_data;
+    b2Body* body = (b2Body*)user_data;
```
Comment (danimtb) [inline src/Game.cpp:16]:
```suggestion
```

**Pull request closed without merging**